app/src/main/java/org/freedombox/freedombox/utils/storage/sharedPreference.kt

```diff
@@ -0,0 +1,35 @@
+/*
+ * This file is part of FreedomBox.
+ *
+ * FreedomBox is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * FreedomBox is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with FreedomBox. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.freedombox.freedombox.utils.storage
+
+import android.content.SharedPreferences
+import android.content.res.Resources
+import com.google.gson.Gson
+import org.freedombox.freedombox.views.model.ConfigModel
+
+fun getSharedPreference(sharedPreferences: SharedPreferences, key: String)
+        = sharedPreferences.getString(key, null)
+
+fun putSharedPreference(sharedPreferences: SharedPreferences,
+                        key: String,
+                        listItem: List<ConfigModel>) {
+    val sharedPreferencesEditor = sharedPreferences.edit()
+    sharedPreferencesEditor.putString(key,
+            Gson().toJson(listItem))
+    sharedPreferencesEditor.apply()
+}
```

Comment:
Ensure a blank line at EOF. There is an IntelliJ setting which ensures this.

Reply:
I am trying to make that change in Intellij settings. Now I think it should be able to add automatically.